CHANGELOG.md

```diff
@@ -1,5 +1,208 @@
 # Changelog
 
+## 5.0.0.rc1
+
+This is a major Teleport release with a focus on new features, functionality, and
+bug fixes. It’s a substantial release and users can review [5.0 closed issues](https://github.com/gravitational/teleport/milestone/39?closed=1)
+on Github for details of all items.
+
+#### New Features
+Teleport 5.0 is a major version and an introduction to Teleports Unified Access
+Plane. Going beyond just SSH access, with first class Kubernetes support
+and the new addition of Application Support.
+
+##### Teleport Application Access
+Teleport can be used to provide secure access to Applications. We've
+built this Teleport Application Access with an eye for securing those internal apps
+which might once of lived on the VPN or have a simple oauth mechanism but with little
+to no audit trail.  We've tested everything from router control panels to Single
```

Comment:
why Single is upper case? Great sentence btw

Reply:
Just a typo - Fixed.